crates/extensions/tedge_config_manager/src/actor.rs

```diff
@@ -97,20 +100,30 @@ impl ConfigManagerActor {
         plugin_config: PluginConfig,
         input_receiver: LoggingReceiver<ConfigInput>,
         output_sender: LoggingSender<ConfigOperationData>,
-        download_sender: DynSender<ConfigDownloadRequest>,
-        upload_sender: DynSender<ConfigUploadRequest>,
+        download_sender: ClientMessageBox<ConfigDownloadRequest, ConfigDownloadResult>,
+        upload_sender: ClientMessageBox<ConfigUploadRequest, ConfigUploadResult>,
```

Comment:
Also you can rename both to `downloader` and `uploader`

Reply:
fixed